### PR TITLE
Implements zfs usage by returning the USED field from the zfs snapshot.

### PR DESCRIPTION
**Pull request by jordanhatcher**
Attempt at an implementation for #17

Signed-off-by: Jordan Hatcher <jordan@jordan-hatcher.com>

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Thanks for working on this, could you squash commits?

Also, can we have a unit test? (for both KindActive and KindCommitted)

File: zfs.go [hunk 1]
```diff
@@ -116,7 +117,39 @@ func (z *snapshotter) Stat(ctx context.Context, key string) (snapshots.Info, err
...
-	return snapshots.Usage{}, errors.New("zfs does not implement Usage() yet")
+	return z.usage(ctx, key)
+}
+
+func (z *snapshotter) usage(ctx context.Context, key string) (snapshots.Usage, error) {
+	ctx, t, err := z.ms.TransactionContext(ctx, false)
+	if err != nil {
+		return snapshots.Usage{}, err
+	}
+	id, info, usage, err := storage.GetInfo(ctx, key)
```
Comment (fuweid) [inline zfs.go:128]: should add `t.Rollback()` here :)

**Review by fuweid — COMMENTED** [review]
could you mind to squash your commits? thanks

**Review by AkihiroSuda — APPROVED** [review]
LGTM, thanks for working on this!

**Review by fuweid — APPROVED** [review]
LGTM

**Pull request merged**